docs/1.x/api/addresses.md

```diff
@@ -58,18 +72,26 @@ Let's take an example of creating an address,
 
 ::: details Response
 
-~~~json
-{
-    "message":"Address has been created successfully.",
-    "data": {...} // This contains the data you send to the api.
-}
-~~~
+  ~~~json
+  {
+      "message":"Address has been created successfully.",
+      "data": {...} // This contains the data you send to the api.
+  }
+  ~~~
+
 :::
 
 ## Get all addresses
 
 To get all the address of a customer, that customer must be logged in to the store. You can achieve this job by using `addresses` API call resource.
```

Comment:
To get all the addresses of a customer, that customer must be logged in to the store. You can achieve this job by using `addresses` API call resource.